tests/ui/feature-gates/feature-gate-return_type_notation.rs

```diff
@@ -1,21 +1,13 @@
 //@ edition: 2021
 //@ revisions: cfg no
 
-//@ [no] check-pass
-// Since we're not adding new syntax, `cfg`'d out RTN must pass.
```

Comment:
This is no longer true since `(..)` is syntactically distinct from paren sugar args (e.g. `Fn()`), so we can deny it pre-expansion too.